src/app/performance-display/models/SuccessMessage.ts

```diff
@@ -0,0 +1,5 @@
+export class ActivitySuccessObject {
+  currentActivityName: string;
+  nextActivityURL: string;
+  nextActivity: string;
+}
```

Comment:
new line

Reply:
@Raxerz by newline do we mean a new empty line after the brace? If that's the thing, then I have done that.

Reply:
Done.